packages/system/kamaji/values.yaml

```diff
@@ -3,7 +3,7 @@ kamaji:
     deploy: false
   image:
     pullPolicy: IfNotPresent
-    tag: v0.25.3@sha256:229646a728b58dd0c55dae7abd721ab23e3feecd61f55fa3ad24bb3a614d558f
+    tag: latest@sha256:04aaf9acb6f49e0315295cd0c24655f28b74c6cf628fc6721758842a41d2deef
```

Comment:
Please omit these changes, version will be automatically bumped in release cycle